btclightclient/btc_header_bytes.go

```diff
@@ -3,18 +3,29 @@ package btclightclient
 import (
 	"bytes"
 	"encoding/hex"
+	"errors"
 
 	"github.com/btcsuite/btcd/wire"
 )
 
 // We don't want to make it compilicated yet :D
 // type BTCHeaderBytes []byte
-// const BTCHeaderSize = 80
+
+const BTCHeaderSize = 80
 
 func BlockHeaderFromHex(hexStr string) (wire.BlockHeader, error) {
-	data, _ := hex.DecodeString(hexStr)
 	var header wire.BlockHeader
+
+	if len(hexStr) != BTCHeaderSize*2 {
+		return header, errors.New("invalid header size, must have 80 bytes")
```

Comment:
```suggestion
		return header, errors.New("invalid header size, must be 80 bytes")
```

Reply:
maybe lets add a new error so we can compare the errors in the test rather than strings